doc/customize.md

```diff
@@ -37,7 +36,10 @@ class CustomUserAgentPlugin implements Plugin
     }
 }
 
-$githubClient->addPlugin(new CustomUserAgentPlugin());
+$httpBuilder = new Github\HttpClient\Builder(new Http\Adapter\Guzzle6\Client());
+$httpBuilder->addPlugin(new CustomUserAgentPlugin());
+
+$client = new Github\Client(null, $httpBuilder);
```

Comment:
`Client::createFromHttpClient($guzzle)`? Otherwise I would go for `createWith..`